server/manager/src/main/java/org/apache/accumulo/manager/upgrade/Upgrader12to13.java

```diff
@@ -64,6 +68,17 @@ public void upgradeMetadata(ServerContext context) {
     addHostingGoalToUserTables(context);
     deleteExternalCompactionFinalStates(context);
     deleteExternalCompactions(context);
+    removeMetaDataBulkLoadFilter(context, MetadataTable.NAME);
+  }
+
+  private void removeMetaDataBulkLoadFilter(ServerContext context, String tableName) {
+    final String propName = Property.TABLE_ITERATOR_PREFIX.getKey() + "majc.bulkLoadFilter";
+    try {
+      context.tableOperations().removeProperty(tableName, propName);
```

Comment:
This may not work during upgrade because it may try to call the client thrift service on the manager which will not start until after upgrade.  May need to go directly to ZK and remove it. Not completely sure if its a problem though.